CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# dbt_recharge_source v0.3.1
+[PR #15](https://github.com/fivetran/dbt_recharge_source/pull/15) includes the following updates:
+## Bugfix
+- Updates the `is_most_recent_record` field in `stg_recharge__subscription_history` to partition on `id`, or `subscription_id` if the former is not available in the source table.
```

Comment:
I would add an explanation of why this had to be done.

Reply:
Added!